perf/perf_stress_ng.py

```diff
@@ -243,10 +243,12 @@ def tearDown(self):
         removes the log files and collects the dmesg data
         :param: none
         """
-
-        file_name = "rm -rf /tmp/stdout /tmp/stderr /tmp/data* /tmp/stressng_output*"
-        try:
-            os.remove(file_name)
-        except OSError:
-            self.log.warn("Files do not exist")
+        files_to_remove = ["/tmp/stdout", "/tmp/stderr", "/tmp/data*",
+                           "/tmp/stressng_output*"]
+        for file_path in files_to_remove:
+            if os.path.exists(file_path):
```

Comment:
Since now a check is added for file exists, there is no need of try can catch correct?
The warning can be dropped to simplify the code

for file_path in files_to_remove:
            if os.path.exists(file_path):
                 os.remove(file_path)

In fact there was no need to add this warning in first place.